### PR TITLE
Add edges

### PR DESCRIPTION
**Pull request by pacmanboss256**
Add a command for "edges"

### CODE REVIEW

File: shared/src/vyxal/Elements.scala [hunk 1]
```diff
@@ -1002,6 +1002,19 @@ object Elements:
...
+      "øh",
+      "Ends",
+      List("ends", "sides", "edges"),
+      false,
+      "a: lst -> [a[0], a[-1]]",
+      "a: str -> [a[0], a[-1]]",
+    ) { a =>
+      val iterable = ListHelpers.makeIterable(a)
+      if iterable.isEmpty then VList.from(Seq.empty)
+      else VList(iterable.tail, iterable.head)
```
Comment (ysthakur) [inline shared/src/vyxal/Elements.scala]:
```suggestion
      else VList(iterable.head, iterable.last)
```
`a.tail` is equivalent to Python's `a[1:]`

**Review by ysthakur — CHANGES_REQUESTED** [review]

File: shared/test/resources/tests.yaml [hunk 2]
```diff
@@ -1501,3 +1501,9 @@
 "Þi":
   - {in: [[1,[2,3]],[1,0]], out: 2}
   - {in: [["xyzabc"], [0,4]], out: "b"}
+
+  "øh":
+  - {in: [1,2,3,4], out: [1, 4]}
+  - {in: [[1,2], 3, 4], out: [[1, 2], 4]}
+  - {in: "hello", out: ["h", "o"]}
+  - {in: 12431, out: [1,1]}
```
Comment (ysthakur) [inline shared/test/resources/tests.yaml]: Hmm, what's the behavior on complex numbers?

Comment (pacmanboss256) [reply shared/test/resources/tests.yaml]: ill figure it out and get back to you

File: shared/test/resources/tests.yaml [hunk 3]
```diff
@@ -1501,3 +1501,9 @@
 "Þi":
   - {in: [[1,[2,3]],[1,0]], out: 2}
   - {in: [["xyzabc"], [0,4]], out: "b"}
+
+  "øh":
```
Comment (ysthakur) [inline shared/test/resources/tests.yaml]:
```suggestion
"øh":
```

File: shared/test/resources/tests.yaml [hunk 4]
```diff
@@ -1501,3 +1501,9 @@
 "Þi":
   - {in: [[1,[2,3]],[1,0]], out: 2}
   - {in: [["xyzabc"], [0,4]], out: "b"}
+
+"øh":
+  - {in: [1,2,3,4], out: [1, 4]}
+  - {in: [[1,2], 3, 4], out: [[1, 2], 4]}
+  - {in: "hello", out: ["h", "o"]}
+  - {in: 12431, out: [1,1]}
```
Comment (ysthakur) [inline shared/test/resources/tests.yaml]:
```suggestion
  - {in: [[1,2,3,4]], out: [1, 4]}
  - {in: [[[1,2], 3, 4]], out: [[1, 2], 4]}
  - {in: ["hello"], out: ["h", "o"]}
  - {in: [12431], out: [1,1]}
```

File: shared/src/vyxal/Elements.scala [hunk 5]
```diff
@@ -1002,6 +1002,19 @@ object Elements:
           ctx.push(s.dropRight(1), if s.isEmpty then "" else s.last.toString)
         case arg => throw UnimplementedOverloadException("ṫ", List(arg))
     },
+    addFull(
+      Monad,
+      "øh",
```
Comment (lyxal) [inline shared/src/vyxal/Elements.scala]: Why is this in the string digraph namespace? I would have thought it'd be better suited for the list digraph namespace (`Þ`)

**Review by lyxal — CHANGES_REQUESTED** [review]

File: shared/src/vyxal/Elements.scala (hunk 5 above)
Comment (pacmanboss256) [reply shared/src/vyxal/Elements.scala]: no idea tbh

Comment (ysthakur) [reply shared/src/vyxal/Elements.scala]:
```suggestion
      "Þh",
```
Then you can call it `Þh`

File: shared/test/resources/tests.yaml [hunk 6]
```diff
@@ -1501,3 +1501,9 @@
 "Þi":
   - {in: [[1,[2,3]],[1,0]], out: 2}
   - {in: [["xyzabc"], [0,4]], out: "b"}
+
+"øh":
```
Comment (ysthakur) [inline shared/test/resources/tests.yaml]:
```suggestion
"Þh":
```
One last thing and then I'll approve

**Review by ysthakur — APPROVED** [review]
Thanks (both for working on this and your patience, sorry about the YAML tests giving unhelpful errors)

**Pull request merged**